libs/judicial-system/types/src/lib/feature.ts

```diff
@@ -2,4 +2,5 @@ export enum Feature {
   NONE = 'NONE', // must be at least one
   INDICTMENTS = 'INDICTMENTS',
   INDICTMENT_ROUTE = 'INDICTMENT_ROUTE',
+  APPEAL_TO_COURT_OF_APPEALS = 'APPEAL_TO_COURT_OF_APPEALS',
```

Comment:
I added this here so that the code wouldn't break but the complete feature hiding functionality relies on another branch that hasn't been merged in to main yet, so when that one has been merged I can update this one and the feature hiding will work.

Reply:
https://github.com/island-is/island.is/pull/10582 the PR for the branch Im talking about